nixos/modules/services/misc/stash.nix

```
@@ -0,0 +1,134 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.stash;
+  opt = options.services.stash;
+
+in {
+
+  options.services.stash = {
+    enable = lib.mkEnableOption "Enable Stash service";
+
+    user = lib.mkOption {
+      type = lib.types.str;
```

```suggestion
    user = mkOption {
      type = types.str;
```

Related to https://github.com/NixOS/nixpkgs/pull/173799/files#r878747840